### DISCUSSION TITLE
Скрытие поля по условию

### DISCUSSION THREAD

**Discussion by juramebel** [totumonline/totum-ru-issues-and-discussions#142] [Как это сделать?] [answered]
Здравствуйте!
Такой вопрос. 
Никогда не пользовался в setformat hide.
И почему то он не срабатывает. 
То есть в панели работает.  А просто в таблице не скрывается поле.
Моя задача выглядит так.
В документе движения номенклатуры чтобы склад получатель появлялся только при выборе типа документа перемещения.
Блокировка от изменения работает, а само поле не прячется.
Находится оно в хэдере.
Что то делаю не так?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Скрытие поля вынесено в форматирование таблицы, так-как нельзя в колонке скрыть поле только в некоторых строках. Можно только целиком.

Параметр `fieldhide: "field_name" = true` функции `setTableFormat`.

Этот параметр снимает галочку в видимости поля в управлению видимостью полей — в `Глазе`. Сама она обратно не включается — если по другим условиям вам нужно показать поле то надо это указать — `fieldhide: "field_name" = false`.

Есть еще вариант отключить так, что бы пользователь не видел этого поля в `Глазе` — `fieldhide: "field_name" = "force"`, но в коде страницы поле с его значениями все-равно передается, поэтому это не является защищенным способом скрытия информации!
